components/Feature/ActionsList/index.js

```diff
@@ -48,10 +48,11 @@ const ActionsList = ({
               <TableRow key={action.id}>
                 <TableData className={styles['lbh-actions-list__complete']}>
                   <Checkbox
-                    aria-label="Completed?"
+                    aria-label={`completed-${action.id}`}
                     checked={action.isCompleted}
                     data-testid="action-checkbox"
-                    name="completed"
+                    name={`completed-${action.id}`}
+                    label={`completed-${action.summary}`}
```

Comment:
???

Reply:
Changed the wording and changed it to labeledby if that's what you meant?
This aria is for the actual checkbox and references the label of the checkbox